workers/unity/Packages/com.improbable.gdk.core/Config/RuntimeConfig.cs

```diff
@@ -29,4 +31,73 @@ public static class RuntimeConfigNames
         public const string WorkerId = "workerId";
         public const string WorkerType = "workerType";
     }
+
+    /// <summary>
+    ///     Stores the configuration needed to connect via the Lcoator.
+    /// </summary>
+    public struct LocatorConfig
+    {
+        /// <summary>
+        ///     The host to connect to the Locator.
+        /// </summary>
+        public string LocatorHost;
+
+        /// <summary>
+        ///     The parameters needed to connect to the Locator.
+        /// </summary>
+        public LocatorParameters LocatorParameters;
+
+        /// <summary>
+        /// A function that takes as input a list of available deployments and
+        /// returns the name of the deployment that we want to connect to via the Locator.
+        /// </summary>
+        public Func<DeploymentList, string> DeploymentListCallback;
+    }
+
+    /// <summary>
+    ///     Stores the configuration needed to connect via the Alpha Locator.
+    /// </summary>
+    public struct AlphaLocatorConfig
```

Comment:
Possibly better to swap these names around. If we know this one is definitely going to be the one we use going forward it might be better to call this the LocatorConfig and the other one the LegacyLocatorConfig.

Reply:
Or we change the names to say what kind of tokens they are using. We can change this later though so it's not that big a deal.

Reply:
Main reason why I call them that way is because the Worker SDK is calling them like that. The current locator flow is still the standard locator, while the new locator is in the alpha namespace